static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss

```diff
@@ -223,6 +223,9 @@ Show more button:
     @include fs-textSans(1);
     line-height: $gs-baseline * 2 - 2px;
     font-weight: bold;
+    white-space: nowrap;
```

Comment:
From what I've seen other buttons on the site truncate. A two line design looks pretty bad (especially when choosing where to place the arrow). I think as the only thing lost at 320px is the arrow this doesn't compromise usability. This is the longest example of a small cta in a commercial component I've managed to find.